### PR TITLE
identity manager

### PR DESCRIPTION
**Pull request by michaelkirk**
PTAL @charlesmchen

### CODE REVIEW

File: src/Account/TSPreKeyManager.m [hunk 1]
```diff
@@ -127,11 +128,12 @@ + (void)registerPreKeysWithMode:(RefreshPreKeysMode)mode
 
         RefreshPreKeysMode modeCopy = mode;
         TSStorageManager *storageManager = [TSStorageManager sharedManager];
-        ECKeyPair *identityKeyPair = [storageManager identityKeyPair];
+        ECKeyPair *identityKeyPair = [[OWSIdentityManager sharedManager] identityKeyPair];
 
+        // TODO: Pull this into OWSIdentityManager?
```
Comment (charlesmchen) [inline src/Account/TSPreKeyManager.m]: This doesn't make sense; we need to update `modeCopy` too (see below).

**Review by charlesmchen — APPROVED** [review]
LGTM

**Review by michaelkirk — COMMENTED** [review]
LGTM

**Comment by michaelkirk** [comment]
new changes LGTM

**Pull request merged**